File: src/me/RoddyRappaport/addressbook/adbookAllCommand.java
package me.RoddyRappaport.addressbook;

import org.bukkit.*;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

import java.util.*;

public class adbookAllCommand  implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] args) {
        if (!args[0].equalsIgnoreCase("show"))
        {
            if (!commandSender.isOp())
            {
                commandSender.sendMessage(ChatColor.RED + "ERROR: You don't have sufficient privilege to execute command");
                return false;
            }
        }
        return AddressBook.useAddressBook(commandSender, new UUID(0, 0), command, s, args);
    }
}

File: src/main/kotlin/AdbookAllCommand.kt
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import java.util.*

class AdbookAllCommand : CommandExecutor {
    override fun onCommand(commandSender: CommandSender, command: Command, s: String, args: Array<String>): Boolean {
        if (!args[0].equals("show", ignoreCase = true)) {
            if (!commandSender.isOp) {
                commandSender.sendMessage(ChatColor.RED.toString() + "ERROR: You don't have sufficient privilege to execute command")
                return false
            }
        }
        return AddressBook.useAddressBook(commandSender, UUID(0, 0), args)
    }
}